src/screens/courses/cardTemplates/OrderTheSequenceCard/styles.ts

```diff
@@ -3,15 +3,15 @@ import { GREY } from '../../../../styles/colors';
 import { FIRA_SANS_REGULAR } from '../../../../styles/fonts';
 import { MARGIN } from '../../../../styles/metrics';
 
-const styles = (backgroundColor: string) => StyleSheet.create({
+const styles = (backgroundColor: string, isValidated: boolean) => StyleSheet.create({
   safeArea: {
     flex: 1,
     backgroundColor: GREY[100],
   },
   container: {
     marginHorizontal: MARGIN.LG,
     flex: 1,
-    justifyContent: 'flex-end',
+    ...(!isValidated && { justifyContent: 'flex-end' }),
```

Comment:
Pourquoi cet ajout ? Je ne vois pas le pb qu'il y avait a toujours justifier le contenu

Reply:
ça ne s'affiche pas mal sur web, taille d'écran mobile  ? 
![Capture d’écran 2024-05-30 à 14 27 16](https://github.com/sophiemoustard/compani-formation/assets/70574422/918ed507-923f-4629-ba18-f06543a368af)

Reply:
Ah si ! Peux tu detailler ce cas dans la partie bug du ticket dans ce cas ? Car c'est pas evident qu'on devait traiter ce bug dans le ticket

Reply:
oui c'est vrai, je l'ai réglé parce qu'il y a un scroll a cet endroit sur mobile quand on a validé la carte pour pouvoir lire les proposition derrière le bandeau (qui d'ailleurs ne fonctionne pas sur web mais j'ai fait l'impasse dessus parce que ça me semblait pas si grave)

Reply:
update : le drag and drop ne fonctionne plus depuis le passage à react-native-gesture-handler 2.16 dans expo 51. Je propose de régler ça dans un ticket à part et de retirer du coup pour le moment les changements que j'ai fait sur cette carte